docs/modules/ROOT/pages/erc20-flash-mint.adoc

```diff
@@ -0,0 +1,73 @@
+= ERC-20 Flash Mint
+
+Extension of xref:erc20.adoc[ERC-20] that provides flash loan support at the token level.
+
+[[usage]]
+== Usage
+
+In order to make https://docs.rs/openzeppelin-stylus/0.2.0-alpha.2/openzeppelin_stylus/token/erc20/extensions/flash_mint/index.html[`ERC-20 Flash Mint`]  methods “external” so that other contracts can call them, you need to add the following code to your contract:
```

Comment:
```suggestion
In order to make https://docs.rs/openzeppelin-stylus/0.2.0-alpha.3/openzeppelin_stylus/token/erc20/extensions/flash_mint/index.html[`ERC-20 Flash Mint`] methods “external” so that other contracts can call them, you need to add the following code to your contract:
```